index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="container">
+        <header class="header">
+            
+            <div class="nav">
+                <span class="logo">John Doe</span>
```

Comment:
- [ ] This logo should behave like a link to the home page so this will help users to redirect to the home page. Kindly change the HTML tag to an `a` tag.